manifest.md

```diff
@@ -193,16 +201,24 @@ Unlike the [Manifest List](#manifest-list), which contains information about a s
 
 - **`annotations`** *hashmap*
 
-    This OPTIONAL property contains arbitrary metadata for the manifest list.
-    Annotations is a key-value, unordered hashmap.
+    This OPTIONAL property contains arbitrary metadata for the image manifest.
+    Annotations MUST be a key-value, unordered hashmap where both the key and value MUST be strings.
+    While the value MUST be present, it MAY be an empty string.
     Keys MUST be unique within the hashmap, and best practice is to namespace the keys.
+    Keys using the `org.opencontainers` namespace are reserved and MUST NOT be used.
+    If there are no annotations then this property MAY either be absent or be an empty hashmap.
+    Implementations that are reading/processing the image manifest MUST NOT generate an error if they encounter an unknown annotation key.
     Common annotation keys include:
     * **created** date on which the image was built (string, timestamps type)
     * **authors** contact details of the people or organization responsible for the image (freeform string)
     * **homepage** URL to find more information on the image (string, must be a URL with scheme HTTP or HTTPS)
     * **documentation** URL to get documentation on the image (string, must be a URL with scheme HTTP or HTTPS)
     * **source** URL to get the source code for the binary files in the image (string, must be a URL with scheme HTTP or HTTPS)
```

Comment:
It feels strange to claim the `org.opencontainers.` namespace and then define “common” keys.  Are these part of the spec?  Are other folks free to interpret these keys with different semantics?  I'd rather put them in our namespace (following the SHOULD recommendation you removed with 100dcce8f → cf0fcb6b2 and the “best practice” recommendation that remains) and use`org.opencontainers.created`, `org.opencontainers.authors`, etc.